tests.rb

```diff
@@ -213,33 +223,36 @@ def test_delete_lesson_deletes_associated_readings
 # Note:  lessons table has a course_id
 #-------------------------------------------------------------
   def test_course_has_table_columns
-    new_course = Course.create( name: "Course.1" )
+    new_course = Course.create(name: "Course B21", course_code: "BBB021" )
     assert new_course.respond_to?("id?")
     assert new_course.respond_to?("term_id")
+    new_course.destroy
   end
 
   def test_course_has_many_lessons
     new_course = Course.create( name: "Course1", course_code: "COO100" )
     l1 = Lesson.create(course_id: new_course.id, name: "Lesson 1")
     l2 = Lesson.create(course_id: new_course.id, name: "Lesson 2")
-    assert new_course.lessons.count > 1
+    assert new_course.lessons.count == 2
+    new_course.destroy
   end
 
   def test_lesson_belongs_to_course
     # Fails with message NoMethodError: undefined method `course
     # if this is missing:
-    new_course2 = Course.create( name: "Course1" )
+    new_course2 = Course.create(name: "Course B23", course_code: "BBB023" )
     le = Lesson.create(course_id: new_course2.id, name: "Lesson 1")
-    assert new_course2.lessons
+    assert new_course2.lessons.first == le
+    new_course2.destroy
```

Comment:
I kind of like the work you're doing cleaning up after yourself in these ... haven't seen it in many other submissions. :+1: